katran/lib/KatranLbStructs.h

```diff
@@ -39,6 +39,8 @@ constexpr uint32_t kDefaultMonitorPcktLimit = 0;
 constexpr uint32_t kDefaultMonitorSnapLen = 128;
 constexpr unsigned int kDefaultLruSize = 8000000;
 constexpr uint32_t kNoFlags = 0;
+constexpr uint8_t kLocalReal = 2;
+constexpr uint32_t kLocalVip = 32;
```

Comment:
we do not have any other flags defined anywhere aside from bpf/balancer_const.h . seems strange to start do so only for subset of them.